lib/node_modules/@stdlib/stats/base/dists/laplace/ctor/benchmark/benchmark.js

```diff
@@ -81,20 +89,25 @@ bench( pkg+'::get:mu', function benchmark( bm ) {
 
 bench( pkg+'::set:mu', function benchmark( bm ) {
 	var dist;
+	var len;
 	var mu;
 	var b;
 	var y;
 	var i;
 
 	mu = 2.0;
 	b = 3.0;
+	len = 100;
+	y = new Array( len );
```

Comment:
```suggestion
	y = new Float64Array( len );
```

This change applies throughout the PR. Use `Float64Array` instead of `Array`